chapters/en/chapter3/5.mdx

```diff
@@ -7,19 +7,40 @@
     classNames="absolute z-10 right-0 top-0"
 />
 
-That was fun! In the first two chapters you learned about models and tokenizers, and now you know how to fine-tune them for your own data. To recap, in this chapter you:
-
-{#if fw === 'pt'}
-* Learned about datasets in the [Hub](https://huggingface.co/datasets)
-* Learned how to load and preprocess datasets, including using dynamic padding and collators
-* Implemented your own fine-tuning and evaluation of a model
-* Implemented a lower-level training loop
-* Used 🤗 Accelerate to easily adapt your training loop so it works for multiple GPUs or TPUs
-
-{:else}
-* Learned about datasets in the [Hub](https://huggingface.co/datasets)
-* Learned how to load and preprocess datasets
-* Learned how to fine-tune and evaluate a model with Keras
-* Implemented a custom metric
-
-{/if}
+That was comprehensive! In the first two chapters you learned about models and tokenizers, and now you know how to fine-tune them for your own data using modern best practices. To recap, in this chapter you:
```

Comment:
Maybe exchanging 5.mdx and 6.mdx since this section presents a recap of the unit so could be the final section.